tools/ovc/openvino/tools/ovc/convert_impl.py

```diff
@@ -442,19 +431,19 @@ def _convert(cli_parser: argparse.ArgumentParser, args, python_api_used):
             args['input_model'] = args['input_model'][0]
     try:
         model_framework = None
-        inp_model_is_object = input_model_is_object(args['input_model']) if python_api_used else False
 
+        if check_pytorch_model(args):
+            example_inputs = None
+            if 'example_input' in args and args['example_input'] is not None:
```

Comment:
I don't think you can provide tensors using cmd line, and in fact we do not need them if the model already exported with `torch.jit` or `torch.export`

Reply:
we can convert model from disk using `convert_model` for which we can pass `example_input` to trace loaded `TorchScript`